samples/package.json

```diff
@@ -14,7 +14,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "@google-cloud/os-login": "^0.3.2"
+    "@google-cloud/os-login": "0.3.3"
```

Comment:
@JustinBeckwith @callmehiphop @jkwlui any reason to keep the `^` if we're updating this file with each release?

Reply:
Best practices.  People look at these samples to see how they're supposed to build their app.  I wouldn't suggest someone pin the version in most cases, so I wouldn't pin the version in the samples 🤷‍♂